analysis/plotting.py

```diff
@@ -174,7 +181,7 @@ def coverage_growth_plot(self,
                     loc='upper left',
                     frameon=False)
 
-        axes.set(ylabel='Bugs coverage' if bugs else 'Edge coverage')
+        axes.set(ylabel='Bug coverage' if bugs else 'Region coverage')
```

Comment:
nit: 'Code coverage' or 'Code region coverage' might be clearer

Reply:
+1

Reply:
fixed.